docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+Verify the change by looking for a default option value when running ``cci task info <name>``.
+
+.. code-block:: yaml
+
+    $ cci task info run_tests
+    run_tests
+
+    Description: Runs all apex tests
+
+    Class: cumulusci.tasks.apex.testrunner.RunApexTests
+
+    Command Syntax
+
+        $ cci task run run_tests
+
+    Options
+        .
+        .
+        .
+      -o required_org_code_coverage_percent PERCENTAGE
+        Optional
+        Require at least X percent code coverage across the org following the test run.
+        Default: 90
+
+
+Add a Custom Task
+^^^^^^^^^^^^^^^^^
+
+To define a new task for your project, simply add the name of the new task under the ``tasks`` section of your ``cumulusci.yml`` file.
+
+    Example: Let's create a custom task named ``deploy_reports`` that deploys a set of reports stored in your project's unpackaged metadata located in ``unpackaged/config/reports``.
+
+    First, look up the Python class associated with the standard task ``deploy``. From there we can see that the ``deploy`` task has a ``class_path`` value of ``cumulusci.tasks.salesforce.Deploy``.
+
+    Store the task under the ``tasks`` section of the ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    deploy_reports:
+        description: Deploy Reports 
+        class_path: cumulusci.tasks.salesforce.Deploy
+        group: projectName
+        options:
+            path: unpackaged/config/reports
+
+.. note:: Be sure to include the value we retrieved for ``class_path``. It's also best practice to add a common ``group`` attribute to your custom tasks in order to make it easier to see the tasks that are specific to your project when using ``cci task list``.
+
+Congratulations! You created a new custom task in CumulusCI.
+
+For additional examples of custom tasks, visit the `task recipes<TODO>`_ section in the cookbook.
+
+If you want to `write a custom task in Python<TODO>`_ and make it available to other users in the project, add the task under the ``tasks`` section of the ``cumulusci.yml`` file:
+
+.. code-block:: yaml
+
+    tasks:
+        my_new_task:
+            description: Description of the task
+            class_path: tasks.task_file.MyNewTaskClassName
+            group: projectName
+
+The above assumes that your task's class is named ``MyNewTaskClassName`` and exists in the file ``tasks/task_file.py``.
+
+
+Use Variables for Task Options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When you want to reference a specific value within the ``cumulusci.yml`` file, use the ``$project_config`` variable.
+
+    Example: NPSP uses a variable for the project's namespace by setting a value of ``$project_config.project__package__namespace``. This variable is then referenced in the project's custom ``deploy_qa_config`` task where it passes as the value for the ``namespace_inject`` option.
```

Comment:
```suggestion
    Example: NPSP uses a variable for the project's namespace by setting a value of ``$project_config.project__package__namespace``. This variable is then referenced in the project's custom ``deploy_qa_config`` task where it passes as the value for the ``namespace_inject`` option.
```
Would we more typically say "it's passed" than "it passes"?

Reply:
I think @davisagli would know this.

Reply:
@justinvholt - Liane is correct. We would want to use "it's passed" here.